lib/Subscriber/Core/MappingSubscriber.php

```diff
@@ -274,11 +268,19 @@ private function hydrateReferenceField(
                 $this->getDefaultValue($fieldMapping)
             );
 
-            if ($referencedNode) {
-                $accessor->set(
+            try {
+                if ($referencedNode) {
+                    $accessor->set(
                     $fieldName,
-                    $this->proxyFactory->createProxyForNode($document, $referencedNode, $options)
+                    $proxyFactory->createProxyForNode($document, $referencedNode, $options)
                 );
+                }
+            } catch (\Exception $e) {
```

Comment:
Again it doesn't matter what Exceptions. If there was an exception it
means that it could not hydrate the proxy class, which should not
happen. When it *does* happen it is crucial to know on which field it
happened. This can save a lot of time when debugging.

If anybody is *listening* for proxy manager exceptions, then somebody
has already done something very wrong.

On Thu, Apr 28, 2016 at 12:56:29AM -0700, Daniel Rotter wrote:
>    In [1]lib/Subscriber/Core/MappingSubscriber.php:
> 
>  >                  );
>  > +                }
>  > +            } catch (\Exception $e) {
> 
>    What comes from the ProxyManager library? Certain exceptions? I am quite
>    sure there is also a more specific exception for that.
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu-document-manager/pull/66#discussion_r61386693
>    2. https://github.com/sulu/sulu-document-manager/pull/66/files/05556aabdb564b020fa388baf338b54d39353978#r61386693
>    3. https://github.com/sulu/sulu-document-manager/pull/66/files/05556aabdb564b020fa388baf338b54d39353978#r61386693